data/human/human missions.txt

```diff
@@ -7453,3 +7453,775 @@ mission "Adelita 5"
+			`	It's Professor-Doctor Hullen-Freeze. He's standing in front of a some university wing named after him.`
+			choice
+				`	"Yes. I met with him sometime back."`
+					goto truth
+				`	"No. I've never seen him in my life."`
+				`	"I refuse to answer without my lawyer present."`
+			`	"Huh. That's so interesting, because we have footage of you entering the building he works in." She pulls out a small screen that shows you, Tavish and Kormak walking into a building.`
+			`	"Are you sure you didn't see him inside? Perhaps you just passed him in the hall?" She looks at you with an open concerned expression.`
+			choice
+				`	"Oh, that Dr. Hullen-Freeze. Yes I did meet with him."`
+					goto truth
+				`	"I enter lots of buildings. Doesn't mean I talk to everyone in them."`
+				`	"I refuse to answer without my lawyer present."`
+			`	"Maybe you ran into his executive secrety? A Ms. Starla Stearn?" She looks at you hopefully.`
+			choice
+				`	"Okay. Yes, I did meet her, and I did see Dr. Hullen-Freeze."`
+					goto truth
+				`	"Hmm, no. Doesn't ring a bell. Sorry I can't be more help."`
+				`	"I refuse to answer without my laywer present."`
+			`	"You see the reason I ask is she has submitted a signed affidavit saying she met with you, two other men, and that you forced your way past her into Dr. Hullen-Freeze's office." She gives you a helpless expression, "Come on now, Captain <last>, I can't help you if you don't help yourself."`
+			choice
+				`	"Okay, okay. Yes I met her and him, but I didn't force my way anywhere. She led me to his office."`
+					goto truth
+				`	"Sorry. She must have me confused for someone else. I just was in that building to use the restroom."`
+				`	"I refuse to answer without my lawer present."`
+			`	Detective Song rubs the bridge of her nose and sighs. "Fine. You can go. I am afraid you have not been terribly helpful. If you can think of anything that may help our investigation please let us know." She knocks on the door and one of the men opens it.`
+			`	"We're done here." She tells him, and they let you back out in the spaceport.`
+				decline
+			
+			label truth
+			`	"Ah, excellent. I really appreciate the cooperation. Could you tell me exactly what happened when you met with Ms. Starla and Dr. Hullen-Freeze?"`
+			choice
+				`	"Starla led us to his office where we briefly spoke and then left. Dr. Tavish was an old student of his and wanted to visit him."`
+					goto liebaby
+				`	"I thought I was escorting a former student of his to his office, but then Tavish and Kormak went crazy and Kormak killed him. They said they'd come after me next if I said anything."`
+			`	Her notebook is out in a flash. "That's Doctor Julian Tavish, and I don't really have any records on this Kormak individual, what was his full name?"`
+			choice
+				`	"Gluniairn mac Kormakr, he's from the Deep. But I am not sure where exactly."`
```

Comment:
```suggestion
				`	"Gluniairn mac Kormak, he's from the Deep. But I am not sure where exactly."`
```

Reply:
So, it is supposed to be Kormakr, but everyone calls him Kormak.

Reply:
Alright. I had been the last changer on that line, and thought I may have added it accidentally.